packages/algolia/src/index.ts

```diff
@@ -0,0 +1,17 @@
+import algoliasearchfn, {
+  SearchIndex as AlgoliaSearchIndex,
+  SearchClient as AlgoliaSearchClient,
+} from 'algoliasearch';
+import algoliasearchLiteFn, {
+  SearchClient as AlgoliaSearchClientLite,
+} from 'algoliasearch/lite';
+
+export type SearchClient = AlgoliaSearchClient;
+export type SearchClientLite = AlgoliaSearchClientLite;
+export type SearchIndex = AlgoliaSearchIndex;
```

Comment:
Whats the advantage of doing this?

Reply:
You have no dependency on raw algoliasearch.
You can substitute AlgoliaSearchIndex with something else that has same interface :)